### PR TITLE
Remove AppInsights from Error Stack

### PR DESCRIPTION
**Pull request by markwolff**
- Remove Zone artifacts from Error stack shown in console
- Modify error pointer shown in console to inform user that the error was rethrown and originated elsewhere

### CODE REVIEW

File: AutoCollection/CorrelationContextManager.ts [hunk 1]
```diff
@@ -244,6 +244,9 @@ export class CorrelationContextManager {
                 if (stackFrames.length > 3) {
                     if (stackFrames[2].trim().indexOf("at Error.AppInsightsAsyncCorrelatedErrorWrapper") === 0) {
                         stackFrames.splice(2, 1);
+                    } else if (stackFrames[1].trim().indexOf("at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError") === 0
```
Comment (OsvaldoRosado) [inline AutoCollection/CorrelationContextManager.ts]: Was this logic previously covered by unit tests? If so, extra cases should be added for this.

**Comment by markwolff** [comment]
@OsvaldoRosado Do we want to drop this PR if considering zone --> cls?

**Comment by OsvaldoRosado** [comment]
@markwolff I think this will be useful in the meantime before that change happens, especially since it's a fairly trivial change with a lot of positive impact.

**Comment by markwolff** [comment]
@GuyHarwood just published 1.0.8

**Pull request merged**